registration/serializers.py

```diff
@@ -76,6 +76,9 @@ class FamilyDetailSerializer(serializers.HyperlinkedModelSerializer):
     parent = StudentSerializer()
     children = StudentSerializer(many=True)
     guests = StudentSerializer(many=True)
+    enrolled = SerializerMethodField()
+    current_class = SerializerMethodField()
+    status = SerializerMethodField()
```

Comment:
to avoid repeating code, can we make these to properties on the Family model instead (similarly to `phone_number` in `FamilySerializer`), and update the `FamilySerializer` to reference the properties as well?

Reply:
ya good idea